src/js/jsx/PanelSet.jsx

```diff
@@ -227,6 +227,16 @@ define(function (require, exports, module) {
                 propertiesPanel = nextState[components.PROPERTIES_COL];
             }
 
+            if (modifierState.command) {
```

Comment:
You probably want to use `swapModifier` instead. Also, I think the logic is incorrect. If the swap modifier is present, then _each_ panel's state is swapped.